### ISSUE TITLE
Test failures on fresh clone

### ISSUE THREAD

**Issue by billkidwell** [awslabs/iot-device-simulator#8]
I cloned master, followed the instructions for running the tests, and I am getting failures with no code modifications.  

I am running Node.js version 8.15.0.  It is running on Ununtu 18.04.1 LTS on the Windows Subsystem for Linux.  

Failure information
------------------------------------------------------------------------------
[Test] Services - Device
------------------------------------------------------------------------------
```
  73 passing (2s)
  2 failing

1) DeviceManager
       #updateDevice
         should return device when operation exists, action queued and ddb put successful:
     Error: done() invoked with non-Error: Error occurred while attempting to send action request to simulator queue.
      at _deviceManager.updateDevice.then.catch (lib/deviceManager.spec.js:860:17)
      at <anonymous>
      at runMicrotasksCallback (internal/process/next_tick.js:122:5)
      at _combinedTickCallback (internal/process/next_tick.js:132:7)
      at process._tickDomainCallback (internal/process/next_tick.js:219:9)

  2) DeviceManager
       #updateDevice
         should return error information when operation exists, action queued and ddb put fails:
     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/c/projects/iot-device-simulator/source/services/device/lib/deviceManager.spec.js)
```  
------------------------------------------------------------------------------
[Test] Resources - Helper
------------------------------------------------------------------------------
``` 
 7 passing (42ms)
  1 failing

  1) iotHelper
       #getIotEndpoint
         should return endpoint address when describeEndpoint successful:
     AssertionError: expected 'abcxyz' to deeply equal { endpointAddress: 'abcxyz' }
      at _helper.getIotEndpoint.then (lib/iot-helper.spec.js:31:38)
      at <anonymous>
```
The full test output is attached.

**Comment by billkidwell** [comment]
[test-output.txt](https://github.com/awslabs/iot-device-simulator/files/2856250/test-output.txt)

**Comment by shsenior** [comment]
@billkidwell Thanks for the feedback. We are in the process of getting version 2 out the door which should resolve these issues. In the meantime, you can address the issue for:
[Test] Resources - Helper
source/resources/helper/lib/test-setup.spec.js - change 
`sinon.create.sandbox()` 
to 
`sinon.createSandbox()`

source/resources/helper/lib/iot-helper.spec.js - change 
```
expect(data).to.deep.equal({
    endpointAddress: 'abcxyz'
});
```

to 

`expect(data).to.equal('abcxyz');`

The device api has been rewritten for version 2.0 and unit tests have been validated.

**Comment by shsenior** [comment]
Resolved in version 2.1.0
